molgenis-navigator/src/main/frontend/test/unit/specs/store/store.spec.js

```diff
@@ -125,44 +121,46 @@ describe('store', function () {
           done()
         })
       })
-      it('should set the error is the given package id is not found', function (done) {
+
+      it('should set the error is the given package id is not found', done => {
         const apiResponse = {
           items: [
             {id: 'pack1', label: 'packLabel1'},
             {id: 'pack3', label: 'packLabel3'}
           ]
         }
-        let getSuccess = Promise.resolve(apiResponse)
+
+        const getSuccess = Promise.resolve(apiResponse)
         get.onFirstCall().returns(getSuccess)
 
-        store.dispatch('GET_STATE_FOR_PACKAGE', 'pack2')
-          .catch(function () {
-            expect(store.state.error).to.equal('couldn\'t find package.')
-            done()
-          })
+        store.dispatch('GET_STATE_FOR_PACKAGE', 'pack2').catch(() => {
+          expect(store.state.error).to.equal('couldn\'t find package.')
+          done()
+        })
       })
-      it('should fetch the content for the given packages and build the path', function (done) {
+
+      it('should fetch the content for the given packages and build the path', done => {
         const apiResponse = {
           items: [
             {id: 'level0', label: 'child', parent: {id: 'level1'}},
             {id: 'level1', label: 'parent', parent: {id: 'level2'}},
             {id: 'level2', label: 'grandparent'}
           ]
         }
+
         const entities = [{id: 'e1', label: 'el1'}]
-        let getPackageSuccess = Promise.resolve(apiResponse)
-        let getEntitiesSuccess = Promise.resolve({items: entities})
+        const getPackageSuccess = Promise.resolve(apiResponse)
+        const getEntitiesSuccess = Promise.resolve({items: entities})
         get.onFirstCall().returns(getPackageSuccess)
         get.onSecondCall().returns(getEntitiesSuccess)
 
-        store.dispatch('GET_STATE_FOR_PACKAGE', 'level1')
-          .then(function () {
-            expect(store.state.packages[0].id).to.equal('level0')
-            expect(store.state.entities[0].id).to.equal('e1')
-            expect(store.state.path[0].label).to.equal('grandparent')
-            expect(store.state.path[1].label).to.equal('parent')
-            done()
-          })
+        store.dispatch('GET_STATE_FOR_PACKAGE', 'level1').then(() => {
```

Comment:
Placing the chained call on the next line, seems more clear to me , best not reformat code as part of PR that is focused on solving a specific issue.